libreyolo/models/base/model.py

```
@@ -969,20 +975,43 @@ def track(
+            effective_conf = min(0.1, tracker_config.det_thresh)
+            tracker_obj = OCSortTracker(config=tracker_config)
+        elif tracker == "bytetrack":
+            if tracker_config is None:
+                tracker_config = TrackConfig.from_kwargs(**tracker_kwargs)
+            # ByteTrack needs to see low-confidence detections.
+            effective_conf = tracker_config.track_low_thresh
+            tracker_obj = ByteTracker(config=tracker_config)
+        else:
+            raise ValueError(
+                f"Unknown tracker {tracker!r}; choose 'bytetrack' or 'ocsort'."
+            )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)